src/main/groovy/org/ajoberstar/grgit/auth/AuthConfig.groovy

```diff
@@ -97,11 +99,13 @@ class AuthConfig {
     static final String FORCE_OPTION = 'org.ajoberstar.grgit.auth.force'
     static final String USERNAME_OPTION = 'org.ajoberstar.grgit.auth.username'
     static final String PASSWORD_OPTION = 'org.ajoberstar.grgit.auth.password'
+	static final String PASSPHRASE_OPTION = 'org.ajoberstar.grgit.auth.passphrase'
     static final String SSH_PRIVATE_KEY_OPTION = 'org.ajoberstar.grgit.auth.ssh.private'
     static final String SSH_SESSION_CONFIG_OPTION_PREFIX = 'org.ajoberstar.grgit.auth.session.config.'
 
     static final String USERNAME_ENV_VAR = 'GRGIT_USER'
     static final String PASSWORD_ENV_VAR = 'GRGIT_PASS'
+	static final String PASSPHRASE_ENV_VAR = 'GRGIT_PASSPHRASE'
```

Comment:
I will remove it as we do not have the possibility to give private key by env var.